README.md

```diff
@@ -140,9 +143,34 @@ Ein Codeblock, welcher bestimmte Abläufe vor und nach dem eigenen Code laufen l
 
 ### Common Environments
 
-* [itemize/enumerate](https://en.wikibooks.org/wiki/LaTeX/List_Structures)
-* [table/tabular](https://en.wikibooks.org/wiki/LaTeX/Tables)
-* [figure](https://en.wikibooks.org/wiki/LaTeX/Floats,_Figures_and_Captions)
+- [itemize/enumerate](https://en.wikibooks.org/wiki/LaTeX/List_Structures)
+- [table/tabular](https://en.wikibooks.org/wiki/LaTeX/Tables)
+- [figure](https://en.wikibooks.org/wiki/LaTeX/Floats,_Figures_and_Captions)
+
+## ZPMs Fontvorschlag
```

Comment:
```suggestion
## ZPMs Fontvorschlag
```

Ich würd den Abschnitt definitiv in der readme lassen, aber vielleicht wäre nur `Fonts` eine bessere überschrift. Ist ja nun kein Vorschlag mehr  :^)

Reply:
Achja, da war ja was :D